README.md

```diff
@@ -47,12 +47,12 @@ The application automatically points to your Aeternity Node and AE Channel servi
 
 In case of questions, issues and featur requests, please turn to https://forum.aeternity.com/ .
 
-3. You can run the test on various environments: mainnet, testnet or locally as an independent node mining on its own. You must configure your ae-channel-service properly so it can sign your transactions for you. Sadly, the demo client does not do that so you must fill your key pair here. If you’re running a local standalone test, you must configure your accounts_test.json accordingly.
+3. You can run the test on various environments: mainnet, testnet or locally as an independent node mining on its own. For each of the environmeents, you must configure your ae-channel-service properly so it can sign your transactions for you.
```

Comment:
```suggestion
3. You can run the test on various environments: mainnet, testnet or locally as an independent node mining on its own. For each of the environments, you must configure your ae-channel-service properly so it can sign your transactions for you.
```